requirements/prod.in

```diff
@@ -13,15 +13,15 @@ django-js-reverse==0.9.1
 django-log-request-id==1.7.0
 django-model-utils==4.2.0
 django-parler==2.3
-django-redis==5.0.0
+django-redis==5.1.0
 django-rq==2.5.1
 djangorestframework==3.12.4
 drf-spectacular==0.21.0
 furl==2.1.3
 github3-py==2.0.0
 honcho==1.1.0
 logfmt==0.4
-markdown==3.3.6
+markdown==3.3.4
```

Comment:
@davisagli I had to downgrade markdown because of cumulusci's strict requirement of `importlib-metadata==4.2.0`, which conflicted with more recent version of markdown that require `importlib-metadata>=4.4`. Probably not a big deal, but I wanted to note it.

Reply:
We should be able to clean this up soon since CumulusCI is dropping support for Python 3.6, which is where flake8 was stubbornly requiring the old importlib-metadata.